pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -61,9 +62,12 @@ public void start() {
         .thenCompose(
             replayDepositsResult ->
                 getHead().thenCompose(headBlock -> processStart(headBlock, replayDepositsResult)))
-        .finish(
-            () -> LOG.info("Eth1DepositsManager successfully ran startup sequence."),
-            (err) -> LOG.fatal("Eth1DepositsManager unable to run startup sequence.", err));
+        .thenAccept(__ -> LOG.info("Eth1DepositsManager successfully ran startup sequence."))
+        .exceptionally(
+            (err) -> {
+              throw new FatalServiceFailureException(getClass(), err);
+            })
```

Comment:
Just to confirm, this doesn't apply when the ETH1 node is down at startup right? We should keep retrying in that case until the node comes up.  Otherwise it gets very difficult to coordinate services properly in things like docker-compose.

Reply:
good question - i'll double-check

Reply:
Tested manually - we keep retrying if the eth1 node isn't up.  Also added some unit tests.